packages/core/content-type-builder/admin/src/components/FormModal/contentType/createContentTypeSchema.js

```diff
@@ -73,6 +85,17 @@ const createContentTypeSchema = ({
           return !reservedModels.includes(value?.trim()?.toLowerCase());
         },
       })
+      .test({
+        name: 'pluralNameNotAlreadyUsedInCollectionName',
+        message: getTrad('error.contentType.pluralName-equals-collectionName'),
+        test(value) {
+          if (!value) {
+            return false;
+          }
+
+          return !collectionNames.includes(value?.trim()?.toLowerCase());
```

Comment:
Why do you trim and lowercase the value? Do the collectionNames come that way? Just curiosity, because maybe it would make sense to clarify this in a comment or something?

Reply:
```suggestion
          return !collectionNames.map(name => name.trim().toLowerCase()).includes(value?.trim()?.toLowerCase());
```

Oh yeah, if the collectionNames are not all normalized to lowercase this will miss some. What we actually need is something like this suggestion (but maybe not exactly this, it's just a quick example)

Reply:
@simotae14  One last thing to check on before merging is the normalized collection names here. Is it possible that collectionNames has uppercase characters in it? Looking at the code where it's set, it looks like it's possible: `return get(contentType, ['schema', 'collectionName'], '');`

Reply:
let me check one more time, otherwise I will remove the toLowerCase

Reply:
yes I can confirm the collectionNames don't contains uppercase characters
`['admin_permissions', 'admin_users', 'admin_roles', 'strapi_api_tokens', 'strapi_api_token_permissions', 'strapi_transfer_tokens', 'strapi_transfer_token_permissions', 'strapi_workflows', 'strapi_workflows_stages', 'files', 'upload_folders', 'myplugin_test', 'i18n_locale', 'up_permissions', 'up_roles', 'up_users', 'addresses', 'categories', 'countries', 'homepages', 'kitchensinks', 'likes', 'menus', 'menusections', 'relation_locales', 'restaurants', 'reviews', 'tags', 'temps', 'strapi_audit_logs']`